Drop after x11/plasma6-libplasma >= 6.1.0 update

CMake Error at CMakeLists.txt:38 (find_package):
  Could not find a configuration file for package "Plasma" that is compatible
  with requested version "6.1".

  The following configuration files were considered but not accepted:

    /usr/local/lib/cmake/Plasma/PlasmaConfig.cmake, version: 6.0.4

--- CMakeLists.txt.orig	2024-06-19 20:33:52 UTC
+++ CMakeLists.txt
@@ -35,8 +35,8 @@ find_package(KF6 ${KF6_MIN_VERSION} REQUIRED COMPONENT
   XmlGui
 )
 
-find_package(Plasma ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR} REQUIRED)
-find_package(PlasmaQuick ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR} REQUIRED)
+find_package(Plasma ${PROJECT_VERSION_MAJOR} REQUIRED)
+find_package(PlasmaQuick ${PROJECT_VERSION_MAJOR} REQUIRED)
 
 find_package(LayerShellQt)
 set_package_properties(LayerShellQt PROPERTIES
